### PR TITLE
Feature/dev 9503

### PR DESCRIPTION
**Pull request by Atash3000**
## Testing Steps
Open Box plot, make you you have data set with multiseries.
1. We need to add more series to see results where box plot chart can render more series. used to be one only.
2. Box plot tooltip issue fixed when user enters custom measure value like "Maximum"
3. Updated "Total" measure into "Count" measure where we show  count of items instead of sum.

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

### CODE REVIEW

File: packages/chart/src/CdcChart.tsx [hunk 1]
```diff
@@ -438,20 +435,11 @@ const CdcChart = ({
...
-
-      const uniqueArray = function (arrArg) {
-        return arrArg.filter(function (elem, pos, arr) {
-          return arr.indexOf(elem) === pos
-        })
-      }
+      const combinedData = filteredData || data
+      let allKeys = combinedData.map(d => d[newConfig.xAxis.dataKey])
+      const groups = _.uniq(allKeys)
+      console.log(groups, 'groups')
```
Comment (adamdoe) [inline packages/chart/src/CdcChart.tsx:441]: remove

File: packages/chart/src/components/EditorPanel/EditorPanel.tsx [hunk 2]
```diff
@@ -816,19 +816,6 @@ const EditorPanel = () => {
     return null
   }
 
-  useEffect(() => {
```
Comment (adamdoe) [inline packages/chart/src/components/EditorPanel/EditorPanel.tsx:819]: good!

File: packages/chart/src/components/LinearChart.tsx [hunk 3]
```diff
@@ -41,6 +41,7 @@ import { useTooltip as useCoveTooltip } from '../hooks/useTooltip'
 import { useEditorPermissions } from './EditorPanel/useEditorPermissions'
 import Annotation from './Annotations'
 import { BlurStrokeText } from '@cdc/core/components/BlurStrokeText'
+import BoxPlotChart from './BoxPlot/BoxPlotX'
```
Comment (adamdoe) [inline packages/chart/src/components/LinearChart.tsx:44]: This import doesn't seem correct

File: packages/chart/src/components/BoxPlot/BoxPlot.tsx [hunk 4]
```diff
@@ -0,0 +1,189 @@
+import React, { useContext } from 'react'
```
Comment (adamdoe) [inline packages/chart/src/components/BoxPlot/BoxPlot.tsx:1]: What is new in this file?

File: packages/chart/src/data/initial-state.js [hunk 5]
```diff
@@ -68,9 +68,6 @@ export default {
   boxplot: {
     plots: [],
     borders: 'true',
-    firstQuartilePercentage: 25,
```
Comment (adamdoe) [inline packages/chart/src/data/initial-state.js:71]: It looks like we can remove these from ts as well

**Review by adamdoe — CHANGES_REQUESTED** [review]
Can you add a story or example file here? When I plot price I'm not seeing additional boxes added to the chart area. I see some changes but I'm not sure if its correct. I added a few additional comments for edits. Feel free to closes those after making updates. Thanks, and great work as always!

File: packages/chart/src/components/BoxPlot/BoxPlot.tsx (hunk 4 above)
Comment (Atash3000) [reply packages/chart/src/components/BoxPlot/BoxPlot.tsx:1]: New box plot added.

File: packages/chart/src/components/LinearChart.tsx (hunk 3 above)
Comment (Atash3000) [reply packages/chart/src/components/LinearChart.tsx]: removed

File: packages/chart/src/CdcChart.tsx (hunk 1 above)
Comment (Atash3000) [reply packages/chart/src/CdcChart.tsx]: removed

File: packages/chart/src/data/initial-state.js (hunk 5 above)
Comment (Atash3000) [reply packages/chart/src/data/initial-state.js:71]: removed

**Review by adamdoe — CHANGES_REQUESTED** [review]
In the storybook example, are you able to include outliers/non outliers so we can see the full example?

**Pull request merged**